test/acceptance/job_test.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Community.Acceptance.JobTest do
+  use Community.AcceptanceCase
+
+  test "editing a job" do
+    job = create(:job)
+
+    navigate_to "/jobs/#{job.id}/edit?token=#{job.token}"
```

Comment:
You can! `my_helper(Community.Endpoint, ...)` :D  So I would `alias Community.Endpoint` in your `AcceptanceCase` so you don't need to fully qualify it for url